src/sentry/utils/snuba.py

```diff
@@ -99,8 +99,11 @@ def query(start, end, groupby, conditions=None, filter_keys=None,
     # we need to fetch the issue definitions (issue -> fingerprint hashes)
     aggregate_cols = [a[1] for a in aggregations]
     condition_cols = [c[0] for c in flat_conditions(conditions)]
-    all_cols = groupby + aggregate_cols + condition_cols
+    all_cols = groupby + aggregate_cols + condition_cols + selected_columns
     get_issues = 'issue' in all_cols
+
+    selected_columns = [col for col in selected_columns if col != 'issue']
```

Comment:
I don't think I quite get why 'issue' is removed here

Reply:
oops, thanks. i thought this wasn't supported initially

Reply:
actually... is it?